src/mlpack/methods/linear_regression/linear_regression_main.cpp

```diff
@@ -108,44 +114,44 @@ PARAM_ROW_OUT("output_predictions", "If --test_file is specified, this "
 PARAM_DOUBLE_IN("lambda", "Tikhonov regularization for ridge regression.  If 0,"
     " the method reduces to linear regression.", "l", 0.0);
 
-static void mlpackMain()
+static void linear_regression(util::Params& params, util::Timers& timer)
```

Comment:
wouldn't writing `BINDING_NAME(util::Params& params, util::Timers& timer)` be better here? just a suggestion, it is completely up to you.

Reply:
I agree---changed to `BINDING_NAME` in 08f9f2e33.